FirebasePerformance/Sources/AppActivity/FPRSessionDetails.m

```diff
@@ -36,12 +36,13 @@ - (instancetype)initWithSessionId:(NSString *)sessionId options:(FPRSessionOptio
 - (FPRSessionDetails *)copyWithZone:(NSZone *)zone {
   FPRSessionDetails *detailsCopy = [[[self class] allocWithZone:zone] initWithSessionId:_sessionId
                                                                                 options:_options];
+  detailsCopy.sessionCreationTime = _sessionCreationTime;
   return detailsCopy;
 }
 
 - (NSUInteger)sessionLengthInMinutes {
   NSTimeInterval sessionLengthInSeconds = ABS([self.sessionCreationTime timeIntervalSinceNow]);
-  return (sessionLengthInSeconds / 60);
+  return (NSUInteger)(sessionLengthInSeconds / 60);
```

Comment:
While this is cleaner code, I don't believe it makes a difference in functionality. The compiler should add an implicit cast to the return type.

Reply:
That is so true Paul. The actual fix is 2 lines above and this is just cleaner code.